pypresence/utils.py

```diff
@@ -8,32 +8,32 @@
 
 # Made by https://github.com/LewdNeko ;^)
 def remove_none(d: dict):
-    for item in d.copy():
-        if isinstance(d[item], dict):
-            if len(d[item]):
-                d[item] = remove_none(d[item])
+    for key in d.keys():
+        if isinstance(d[key], dict):
+            if d[key]:
+                remove_none(d[key])
             else:
-                del d[item]
-        elif d[item] is None:
-            del d[item]
-    return d
+                del d[key]
+        elif d[key] is None:
+            del d[key]
```

Comment:
Creating a copy was intentional, this function should not be altering existing dicts in place.

Reply:
d.copy() is how you create a copy. Dicts are mutable.

Reply:
I am aware, its behaviour is intended and will not change.

Reply:
remove_none is only used twice: in payloads.py and recursively. I edited payloads.py to handle the change. 
What you are doing currently is using extra memory when you can avoid a .copy().